### PR TITLE
Intern dashboard

### PR DESCRIPTION
**Pull request by ashatat**
Relates #160

### CODE REVIEW

File: client/src/components/Pages/InternDashboard/mapPin.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="17" height="21" viewBox="0 0 17 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (RamyAlshurafa) [inline client/src/components/Pages/InternDashboard/mapPin.svg]: moving this to `assets` folder would be better

File: client/src/constants/apiRoutes.js [hunk 2]
```diff
@@ -45,6 +45,9 @@ const API_VERIFY_PROFILE_URL = `${API_PREFIX}${VERIFY_PROFILE_URL}`;
 const ORGS_DASHBOARD = "/organisation-dashboard";
 const API_ORGS_DASHBOARD_URL = `${API_PREFIX}${ORGS_DASHBOARD}`;
 
+const INTERN_DASHBOARD_URL = "/intern-dashboard";
```
Comment (RamyAlshurafa) [inline client/src/constants/apiRoutes.js]: "/intern/dashboard" ?
What do you think?

File: client/src/components/Pages/InternDashboard/Update.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React from "react";
```
Comment (RamyAlshurafa) [inline client/src/components/Pages/InternDashboard/Update.js]: move this to the `Common` please

File: client/src/components/Pages/InternDashboard/mapPin.svg (hunk 1 above)
Comment (ashatat) [reply client/src/components/Pages/InternDashboard/mapPin.svg]: Thanks

done

File: client/src/constants/apiRoutes.js (hunk 2 above)
Comment (ashatat) [reply client/src/constants/apiRoutes.js]: yeah I agree that would be better

File: client/src/components/Pages/InternDashboard/Update.js (hunk 3 above)
Comment (ashatat) [reply client/src/components/Pages/InternDashboard/Update.js]: done

**Pull request merged**